### PR TITLE
Add support for checkbox questions

### LINKED ISSUE #260: Add Checkbox as another question type

### LINKED ISSUE #261: Update answer indices to 1-indexed instead of 0-indexed in the downloaded report

### PR DESCRIPTION
**Pull request by dalmia**
Fixes #260 
Fixes #261

## Summary
- added `checkbox` as another valid question type
- changed data type of `answer` in session answer to `JSONField`
- changed data type of `correct_answer` in question to `JSONField`
- updates the README accompanying a plio's report to make it clearer
- in the report, we were giving correct_answer 1-indexed but session answer as 0-indexed. Made it 1-indexed for both now.

## Test Plan

- [ ] ~Wrote tests~
- [x] Tested locally
- [x] Tested on staging
- [ ] Tested on production
- [x] If changes in DB, update [DB schema](https://docs.google.com/presentation/d/15c45HbHphh4w3guaFV3BgEA5nMKd-4J35JdUKdrpPzo/edit#slide=id.gc8c9ec0b5e_0_0) and BigQuery (staging and prod)

### CODE REVIEW

File: plio/models.py [hunk 1]
```diff
@@ -132,7 +132,7 @@ class Question(SafeDeleteModel):
     )
     text = models.TextField(blank=True, default="")
     options = models.JSONField(null=True)
-    correct_answer = models.TextField(null=True)
+    correct_answer = models.JSONField(null=True)
```
[the remark this answers is not in the document]
Comment (dalmia) [reply plio/models.py:135]: sent on discord DM

File: entries/models.py [hunk 2]
```diff
@@ -61,7 +61,7 @@ class SessionAnswer(SafeDeleteModel):
 
     session = models.ForeignKey(Session, on_delete=models.CASCADE)
     item = models.ForeignKey(Item, on_delete=models.CASCADE)
-    answer = models.TextField(null=True)
+    answer = models.JSONField(null=True)
```
[the remark this answers is not in the document]
Comment (dalmia) [reply entries/models.py:64]: sent on discord DM

File: plio/static/plio/docs/download_csv_README.md [hunk 3]
```diff
@@ -44,7 +44,9 @@ This section will clarify what each of the `.csv` files in the folder contains:
   - `question_type`: the type of question (e.g. `mcq`)
   - `question_text`: the text of the question itself
   - `question_options`: the options for the question
-  - `question_correct_answer`: the correct answer for the given question. For `question_type = mcq`, this represents the index of the correct answer among the `question_options`.
+  - `question_correct_answer`: the correct answer for the given question. For `question_type = mcq`, this represents the index of the correct answer among the `question_options`. Here, `answer = 1` indicates that the first option has been marked as the correct answer.
+
+    For `question_type = checkbox`, it contains a list of indices corresponding to the options which are correct. Here, `answer = [2,3]` indicates that the second and third options have been marked as the correct answers.
```
[the remark this answers is not in the document]
Comment (dalmia) [reply plio/static/plio/docs/download_csv_README.md]: update example to add checkbox to table + add new column of question_type

File: plio/static/plio/docs/download_csv_README.md [hunk 4]
```diff
@@ -44,9 +44,9 @@ This section will clarify what each of the `.csv` files in the folder contains:
...
   - `question_text`: the text of the question itself
   - `question_options`: the options for the question
-  - `question_correct_answer`: the correct answer for the given question. For `question_type = mcq`, this represents the index of the correct answer among the `question_options`. Here, `answer = 1` indicates that the first option has been marked as the correct answer.
+  - `question_correct_answer`: the correct answer for the given question. For `question_type = mcq`, this represents the index of the correct answer among the `question_options`. In this case, `question_correct_answer = 1` indicates that the first option has been marked as the correct answer.
 
-    For `question_type = checkbox`, it contains a list of indices corresponding to the options which are correct. Here, `answer = [2,3]` indicates that the second and third options have been marked as the correct answers.
+    For `question_type = checkbox`, it contains a list of indices corresponding to the options which are correct. In this case, `question_correct_answer = [2,3]` indicates that the second and third options have been marked as the correct answers.
```
Comment (deepansh96) [inline plio/static/plio/docs/download_csv_README.md:49]: This case is not present in the example below no?
Also in the example below, I noticed a typo. MCQ is written as MCC

**Review by deepansh96 — CHANGES_REQUESTED** [review]

**Pull request merged**